subscriptions.go

```diff
@@ -89,6 +91,36 @@ type SubscriptionAPI struct {
 	backOffConf backOffConfiguration
 }
 
+type PageParameters struct {
+	Limit int
+	Offset int
+}
+
+type SubscriptionParameters struct {
+	PageParameters
+	OwningApplication string
+	EventType         string
+}
+
+func (p *SubscriptionParameters) EncodeQueryString() string {
```

Comment:
Too keep the public API minimal, this method should not be exported. Please change the name to `encodeQueryString`.